examples/addlayer.js

```diff
@@ -37,13 +36,45 @@
     // Add a layer to a pre-exiting ol.layer.Group after the LayerSwitcher has
     // been added to the map. The layer will appear in the list the next time
     // the LayerSwitcher is shown or LayerSwitcher#renderPanel is called.
-    overlayGroup.getLayers().push(new ol.layer.Tile({
-        title: 'Countries',
-        source: new ol.source.TileWMS({
-            url: 'http://demo.opengeo.org/geoserver/wms',
-            params: {'LAYERS': 'ne:ne_10m_admin_1_states_provinces_lines_shp'},
-            serverType: 'geoserver'
-        })
-    }));
+
+    url = "https://sampleserver1.arcgisonline.com/ArcGIS/rest/services/Specialty/ESRI_StatesCitiesRivers_USA/MapServer";
```

Comment:
Having tested locally on a HiDPI screen it looks as though the map service only allows a maximum width and height of 2048px. I think we should therefore add a style to `addlayer.html` along the lines of:

    <style type="text/css">
        <!-- Limit the dimensions of the map to avoid requesting
             images too large for the ESRI image service -->
        #map {
            max-width: 800px;
            max-height: 800px;
        }
    </style>